package org.example.exceptions;

import java.util.ArrayList;
import java.util.List;

public class Test1 {
    public static void main(String[] args) {
//        int []a={1,3,4,5};
//        List<Integer> list=new ArrayList<>();
//        list.add(2);
//        list.add(5);
//        list.add(7);
//        list.add(3);
//
//        list.get(5);
//
//       // System.out.println(a[9]);
//        try {
//            System.out.println(a[6]);
//
//        }catch (ArrayIndexOutOfBoundsException e){
//            System.out.println("Array Index-of -bound");
//            e.printStackTrace();
//        }
//        System.out.println("Rest of the code");



  }
}
